test/index.spec.js

```diff
@@ -1,13 +1,12 @@
-const chai = require('chai')
-const chaiHttp = require('chai-http')
+const chai = require('chai');
+const chaiHttp = require('chai-http');
 
-chai.use(chaiHttp)
-const expect = chai.expect
```

Comment:
Changing the code of your teammates to the way you prefer is not the best approach. You can suggest changes with good reasons before actually making them in the code.